Nextcloud Dev settings crashed

### Actual behaviour
- when i try to open the settings the app crash

### Expected behaviour
- 
 
### Steps to reproduce
1. open the app
2. go into settings
3. 


### Environment data
Android version: 9

Device model: Samsung j5 2016

Stock or customized system: LineageOS 16

Nextcloud app version: com.nextcloud.android.beta
Version: 20191206

Nextcloud server version:

### Logs
#### Web server error log
```

************ CAUSE OF ERROR ************

java.lang.RuntimeException: Unable to start activity ComponentInfo{com.nextcloud.android.beta/com.owncloud.android.ui.activity.SettingsActivity}: java.lang.ClassCastException: java.lang.Boolean cannot be cast to java.lang.String
 at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2951)
 at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3086)
 at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:78)
 at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:108)
 at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:68)
 at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1816)
 at android.os.Handler.dispatchMessage(Handler.java:106)
 at android.os.Looper.loop(Looper.java:193)
 at android.app.ActivityThread.main(ActivityThread.java:6718)
 at java.lang.reflect.Method.invoke(Native Method)
 at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
 at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
Caused by: java.lang.ClassCastException: java.lang.Boolean cannot be cast to java.lang.String
 at android.app.SharedPreferencesImpl.getString(SharedPreferencesImpl.java:283)
 at android.preference.Preference.getPersistedString(Preference.java:1650)
 at android.preference.ListPreference.onSetInitialValue(ListPreference.java:295)
 at android.preference.Preference.dispatchSetInitialValue(Preference.java:1560)
 at android.preference.Preference.onAttachedToHierarchy(Preference.java:1328)
 at android.preference.PreferenceGroup.addPreference(PreferenceGroup.java:163)
 at android.preference.PreferenceGroup.addItemFromInflater(PreferenceGroup.java:105)
 at android.preference.PreferenceGroup.addItemFromInflater(PreferenceGroup.java:42)
 at android.preference.GenericInflater.rInflate(GenericInflater.java:490)
 at android.preference.GenericInflater.rInflate(GenericInflater.java:495)
 at android.preference.GenericInflater.inflate(GenericInflater.java:327)
 at android.preference.GenericInflater.inflate(GenericInflater.java:264)
 at android.preference.PreferenceManager.inflateFromResource(PreferenceManager.java:324)
 at android.preference.PreferenceActivity.addPreferencesFromResource(PreferenceActivity.java:1549)
 at com.owncloud.android.ui.activity.SettingsActivity.onCreate(SettingsActivity.java:151)
 at android.app.Activity.performCreate(Activity.java:7144)
 at android.app.Activity.performCreate(Activity.java:7135)
 at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1271)
 at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2931)
 ... 11 more

************ APP INFORMATION ************
ID: com.nextcloud.android.beta
Version: 20191206
Build flavor: versionDev

************ DEVICE INFORMATION ************
Brand: samsung
Device: j5xnlte
Model: SM-J510FN
Id: PQ3A.190801.002
Product: lineage_j5xnlte

************ FIRMWARE ************
SDK: 28
Release: 9
Incremental: eng.exodus.20191113.204659
```

#### Nextcloud log (data/nextcloud.log)
```
Insert your Nextcloud log here
```
**NOTE:** Be super sure to remove sensitive data like passwords, note that everybody can look here! You can use the Issue Template application to prefill some of the required information: https://apps.nextcloud.com/apps/issuetemplate

--- a/src/main/java/com/owncloud/android/ui/activity/SettingsActivity.java
+++ b/src/main/java/com/owncloud/android/ui/activity/SettingsActivity.java
@@ -709,7 +709,11 @@ public class SettingsActivity extends ThemedPreferenceActivity
 
         themePref.setEntries(themeEntries.toArray(new String[0]));
         themePref.setEntryValues(themeValues.toArray(new String[0]));
-        themePref.setSummary(themePref.getEntry().length() == 0 ? DarkMode.LIGHT.name() : themePref.getEntry());
+
+        if (TextUtils.isEmpty(themePref.getEntry())) {
+            themePref.setValue(DarkMode.LIGHT.name());
+            themePref.setSummary(TextUtils.isEmpty(themePref.getEntry()) ? DarkMode.LIGHT.name() : themePref.getEntry());
+        }
 
         themePref.setOnPreferenceChangeListener((preference, newValue) -> {
             DarkMode mode = DarkMode.valueOf((String) newValue);

